client/src/Components/Location/Location.js

```diff
@@ -2,18 +2,45 @@ import React, { Component } from 'react';
 import './Location.css';
 
 class Location extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {wifi: false, privacy: false, quiet: false, foodAndDrink: false};
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChange(event) {
+    this.setState({
+      [event.target.id]: !this.state[event.target.id]
+    });
+  }
+
+  handleSubmit(event) {
+    event.preventDefault();
+  }
+
   render() {
     return (
-      <div>
-        <h1>Location</h1>
-        <div>This is a test.</div>
+      <div ref="outputDiv">
+        <form ref="form" onSubmit={this.handleSubmit}>
+          <label className="item">
+            <input id="wifi" type="checkbox" onChange={this.handleChange} /> Wifi
+          </label>
+          <label className="item">
+            <input id="privacy" type="checkbox" onChange={this.handleChange} /> Privacy
+          </label>
+          <label className="item">
+            <input id="quiet" type="checkbox" onChange={this.handleChange} /> Quiet
+          </label>
+          <label className="item">
+            <input id="foodAndDrink" type="checkbox" onChange={this.handleChange} /> Food and Drink
+          </label>
+          <input type="submit" value="Submit" />
+        </form>
       </div>
     );
   }
-
-  value() {
```

Comment:
What is this for? I only see Jooneil use it to return a String value and use that value to set the value property of an input in his placeholder location file.